packages/manager-react-components/src/components/formatted-date/FormattedDate.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
not really sure of the usefulness of this component, isn't the hook enough ?

Reply:
It's for the type here: `React.FC<`. But yes I can delete this dependance

Reply:
No I mean the whole file not this line